src/components/CohortFacilitatorList.tsx

```
@@ -136,16 +136,28 @@ const CohortLearnerList: React.FC<CohortLearnerListProp> = ({
 
-  // const fetchData = async () => {
-  //   try {
-  //     setInfinitePage(
-  //       (prev) => prev + PAGINATION_CONFIG.INFINITE_SCROLL_INCREMENT
-  //     );
-  //   } catch (error) {
-  //     console.error('Error fetching more data:', error);
-  //     showToastMessage(t('COMMON.SOMETHING_WENT_WRONG'), 'error');
-  //   }
-  // };
+  const fetchData = async () => {
```

ESLint

[error] 'fetchData' is assigned a value but never used.

(no-unused-vars)